src/test/scripts/helm_install.sh

```diff
@@ -59,11 +59,21 @@ if grep -q nfs: ${chartValueFiles} /dev/null; then
     echo This configuration requires a private NFS server, starting...
     nfsServerPodName="${PRODUCT_RELEASE_NAME}-nfs-server"
     startNfsServer "${PRODUCT_RELEASE_NAME}" "${nfsServerPodName}"
-    nfsServerIp=$(kubectl get pods -n $TARGET_NAMESPACE "$nfsServerPodName" -o json | jq -r .status.podIP)
-    if [ -z "$nfsServerIp" ]; then
-      echo NFS server not found.
-      exit 1
-    fi
+
+    for ((try = 0; try < 60; try++)) ; do
+      echo Detecting NFS server IP...
+      nfsServerIp=$(kubectl get pods -n $TARGET_NAMESPACE "$nfsServerPodName" -o json | jq -r .status.podIP)
```

Comment:
Do we have `jq` available on Bamboo?

Reply:
we do, I already have a green branch of that running.